src/build/webpack.daemon.config.js

```diff
@@ -8,16 +8,21 @@ const browser = process.env.BROWSER || "chrome";
 const node_env = process.env.NODE_ENV || "production";
 
 const build_dir = __dirname + "/../../build/" + browser;
+const entry =
+  browser === "chrome"
```

Comment:
I find it odd to see a browser selection like this being writen as "major and others". There's a story coming up for an Edge build. Really, it should be a selection from a list. 

This seems like the time for a switch statement?